DesignDocs.md

```diff
@@ -0,0 +1,65 @@
+# Design Documents
+
+## Purposes of this document
+* To describe what our design documents are for, their structure, and what goes in them.
+* To describe how DSNP Design Documents are produced and where they're published.
+
+## What Is a Design Document?
```

Comment:
Do we need to talk about how this differs from a DIP? The goal is different in that a DIP is a specific change to the DSNP spec whereas this is more about ANY open source code we have. I could also see a design doc "producing" a DIP? Or perhaps this is just better than the DIP process for now?

Reply:
Good question. I'm ignorant of what the team has already outlined about a DIP and related process - please share anything if you can - but @jvasile @kfogel and I are meeting Monday to talk through a visual guide for governance and test cases. I don't know the right answer now other than keep it simple. I'll give a better suggestion on this soon.

Reply:
The way I see it is a Design Doc is more directly related to writing code and not for the DSNP Improvement Proposals, although there's certainly overlap.